integrations/utils.ts

```diff
@@ -286,7 +286,13 @@ export function test(
       }
 
       try {
-        context.exec('pnpm install')
+        // In debug mode, the directory is going to be inside the pnpm workspace
+        // of the tailwindcss package. This means that `pnpm install` will run
+        // pnpm install on the workspace instead (expect if the root dir defines
+        // a seperate workspace). We work around this by using the
+        // --ignore-workspace flag.
+        let ignoreWorkspace = debug && !config.fs['pnpm-workspace.yaml']
```

Comment:
This fixes an unrelated problem when `test.debug` is used without setting up a custom pnpm workspace